### PR TITLE
Automate adding new PR's and moving updated PR's on project board

### PR DESCRIPTION
**Pull request by allisonport-db**
### TODO
1. Set up credentials (requires admin access)
    - Generate an [access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) with `repo` access
    - [Store the token](https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-an-organization) as a secret at the organization level
    - Match the secret name in the `yaml` files with the stored token
2. Exclude internal github usernames from this workflow (how should I determine this "list"?)
   ```yaml
   if: !contains(["username1", "username2", ..., "usernameN"], github.event.actor.login)
   ```

### CODE REVIEW

**Comment by allisonport-db** [comment]
So the `github.event.actor` is whoever triggered the event, so in a new or updated PR that should be the owner of the PR, and in a comment it should be the commenter, but we should double check once it's merged.

**Comment by allisonport-db** [comment]
test comment

File: .github/workflows/updated_pull_request.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    types: [created]
+  pull_request_target:
+    types: [synchronize]
+
+jobs:
+  automate-updated-pull-requests:
+    if: ${{ !contains(["allisonport-db", "dennyglee", "scottsand-db", "tdas", "zsxwing"], github.event.actor.login) }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: alex-page/github-project-automation-plus@2af3cf061aeca8ac6ab40a960eee1968a7f9ce0e
```
Comment (zsxwing) [inline .github/workflows/updated_pull_request.yaml:14]: why this is not using a version?

**Review by zsxwing — APPROVED** [review]
Since this has to be merged into the master, feel free to merge it. This doesn't affect any code.

File: .github/workflows/updated_pull_request.yaml (hunk 1 above)
Comment (allisonport-db) [reply .github/workflows/updated_pull_request.yaml:14]: There was a bug in `github-project-automation-plus` so it wouldn't work for comments on PR's (only on issues) that I fixed in a PR. It hasn't been merged / rereleased yet so this points to the commit.

Comment (zsxwing) [reply .github/workflows/updated_pull_request.yaml:14]: I see. Let's add a comment and a TODO item to use a released version.

**Pull request merged**